Gemfile

```diff
@@ -3,11 +3,14 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'vagrant', git: 'https://github.com/mitchellh/vagrant.git'
+  gem 'vagrant',      git: 'https://github.com/mitchellh/vagrant.git'
+  gem 'vagrant-spec', git: 'https://github.com/mitchellh/vagrant-spec.git'
   gem 'gem-compare'
   gem 'rake'
+  gem 'vagrant-vbguest'
 end
 
 group :plugins do
   gem 'vagrant-registration', path: '.'
+  gem 'pry-byebug'
```

Comment:
It's a def dependency. Happy to remove it though, since it is hard do step into vagrant-spec execution.